_pytest/terminal.py

```diff
@@ -42,6 +42,9 @@ def pytest_addoption(parser):
                      action="store", dest="tbstyle", default='auto',
                      choices=['auto', 'long', 'short', 'no', 'line', 'native'],
                      help="traceback print mode (auto/long/short/line/native/no).")
+    group._addoption('--no-stdout',
```

Comment:
Currently we have `--no-print-logs`:

```
  --no-print-logs       disable printing caught logs on failed tests.
```

So perhaps we might also have a `--no-print-capture`:

```
  --no-print-capture      disable printing captured stdout/stderr on failed tests.
```

Or perhaps:

```
--show-capture=no/stdout/stderr/both    controls how captured stdout/stderr is shown on failed tests. default is "both".
```

@The-Compiler @RonnyPfannschmidt @blueyed what's your opinion? I'm leaning on the second one because it seems more flexible.

Reply:
the second one is a good point for me :)

Reply:
I like `--show-capture=no/stdout/stderr/both`.

Reply:
@nicoddemus 
Ok, I think I can try to fix. Is it good place for fix?
https://github.com/pytest-dev/pytest/blob/89a55d85a9d30ef2a2394b3969fc7067f1c15b59/_pytest/terminal.py#L623-L629

or here?
https://github.com/pytest-dev/pytest/blob/89a55d85a9d30ef2a2394b3969fc7067f1c15b59/_pytest/capture.py#L131-L142
In this case I don't know how to send command option from terminal.py there

Reply:
I think applying the change in terminal.py is the way to go, because other plugins (--junitxml for example) might be using `rep.sections`.

Reply:
Of course we will need more tests for each variant now. 😁